src/components/Layout/Sidebar/index.tsx

```diff
@@ -124,6 +132,14 @@ export const SidebarUi: React.FC = () => {
                     >
                       {t(i18nKey)}
                     </Typography>
+
+                    {isNew && (
+                      <div css={styles.listItemNewBadge}>
```

Comment:
Would it make sense to use our chip component here?

Reply:
Same as above